activerecord/test/cases/statement_cache_test.rb

```diff
@@ -112,29 +112,29 @@ def test_unprepared_statements_dont_share_a_cache_with_prepared_statements
     end
 
     def test_find_by_does_not_use_statement_cache_if_table_name_is_changed
-      book = Book.create(name: "my book")
```

Comment:
Maybe inline model would be easier to use in here, since there will be no need for cleanup. :thinking: